### ISSUE TITLE
app.setAppLogsPath() does not set the `logs` path

### ISSUE THREAD

**Issue by felixrieseberg** [electron/electron#20941]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:** 7.0.0, 7.0.0-beta.4
* **Operating System:** macOS 10.14.6 (Mojave)
* **Last Known Working Electron version:** 7.0.0-beta.5

### Expected Behavior
`app.setAppLogsPath(path)` should change the default logging path

### Actual Behavior
`app.setAppLogsPath(desktopPath)` seems to have no effect, logs are always created in `~/Library/Logs/[appName]`

### To Reproduce
Fiddle: https://gist.github.com/eb7c5c66066afba3ab6dbb0353f79657

```js
const {app} = require('electron')
const path = require('path')

app.on('ready', () => {
  const desktopPath = app.getPath(`desktop`)
  app.setAppLogsPath(desktopPath)

  const logPath = app.getPath('logs')
  console.assert(logPath === desktopPath, `Expected ${logPath} to equal ${desktopPath}`);

  app.quit();
})
```

**Comment by felixrieseberg** [comment]
I suspect this PR: https://github.com/electron/electron/pull/20117

**Comment by MarshallOfSound** [comment]
Fixed in #20985
